runtime/themes/darcula.toml

```diff
@@ -0,0 +1,90 @@
+# Author : Nick Ogden <nick@nickogden.org>
+
+"ui.background" = { bg = "grey01" }
+"ui.menu" = { fg = "grey05", bg = "grey00" }
+"ui.menu.selected" = { fg = "grey01", bg = "grey04" }
+"ui.linenr" = { fg = "grey03", bg = "grey01" }
+"ui.linenr.selected" = { fg = "grey04", bg = "grey01", modifiers = ["bold"] }
+"ui.gutter" = { bg = "grey01" }
+"ui.popup" = { fg = "grey05", bg = "grey00" }
+"ui.window" = { bg = "grey01" }
+"ui.selection" = { bg = "grey03" }
+"ui.statusline" = { fg = "grey04", bg = "grey02" }
+"ui.statusline.insert" = { bg = "white", fg = "grey01" }
+"ui.statusline.select" = { bg = "orange", fg = "grey01" }
+"ui.help" = { fg = "grey04", bg = "grey01" }
+"ui.cursor" = { fg = "grey04", modifiers = ["reversed"] }
+"ui.cursor.primary" = { fg = "grey05", modifiers = ["reversed"] }
+"ui.cursor.match" = { fg = "white", modifiers = ["underlined"] }
+"ui.cursorline.primary" = { bg = "grey02" }
+"ui.cursorline.secondary" = { bg = "grey02" }
+"ui.text" = "white"
+"ui.text.focus" = "grey05"
+"ui.virtual.ruler" = { bg = "grey02" }
+"ui.virtual.indent-guide" = "grey02"
```

Comment:
You may want to add a `ui.virtual.whitespace` key (`:set whitespace.render all`). Without it the whitespace looks pretty bright - I think it would look good if it were slightly brighter than the indent guides color. What do you think?

Reply:
I've  now added the whitespace styling as requested. I don't really have an opinion on it as it's a feature that I've never needed myself, but I've gone for the next shade lighter than the indent guides. It looks fine to me.